Domains/4-Application/DrawingProduction/SheetNumberingViews.ecschema.xml

```diff
@@ -0,0 +1,184 @@
+                        -- Start with root nodes-SheetIndexes
+                        SELECT
+                        si.ECInstanceId,
+                        si.ECClassId,
+                        COALESCE([si].[CodeValue], [si].[UserLabel]) AS [SheetNumber],
+                        0 AS [Priority],
+                        NULL AS [DirectParentId],
+                        COALESCE([si].[CodeValue], [si].[UserLabel]) AS [AssemblyPath],
+                        -- Use string padding for consistent sorting - pad to 10 digits with leading zeros
+                        SUBSTR('0000000000' || CAST(0 AS TEXT), -10) AS [SortPath]
+                        FROM bis:SheetIndex si
+
+                        UNION ALL
+                        -- Recursively add children to parents, building hierarchical sort paths
+                        SELECT
+                        child.ECInstanceId,
+                        child.ECClassId,
+                        child.SheetNumber,
+                        child.NodePriority AS [Priority],
+                        child.NodeParentId AS [DirectParentId],
+                        parent.AssemblyPath || '->' || child.SheetNumber AS [AssemblyPath],
+                        parent.SortPath || '.' || SUBSTR('0000000000' || CAST(COALESCE(child.NodePriority, 0) AS TEXT), -10) AS [SortPath]
+                        FROM treeTraversal parent
+                        JOIN (
+                        -- A table of all possible children joined to their parents
+                            SELECT
+                                [sr].[ECInstanceId],
+                                ECClassId,
+                                COALESCE([sr].[CodeValue], [sr].[UserLabel]) AS [SheetNumber],
+                                [sr].[EntryPriority] AS [NodePriority],
+                                [sr].[Parent].[Id] AS [NodeParentId]
+                            FROM
+                                [bis].[SheetReference] [sr]
+
+                            UNION ALL
+
+                            SELECT
+                                [sif].[ECInstanceId],
+                                ECClassId,
+                                COALESCE([sif].[CodeValue], [sif].[UserLabel]) AS [SheetNumber],
+                                [sif].[EntryPriority] AS [NodePriority],
+                                [sif].[Parent].[Id] AS [NodeParentId]
+                            FROM
+                                [bis].[SheetIndexFolder] [sif]
+                        ) child ON child.NodeParentId = parent.ECInstanceId
+                    )
+                    -- Final result: Returns only SheetReference node due to inner join
+                    SELECT
+                        sr.ECInstanceId AS [ECInstanceId],
+                        ec_classid('SheetNumbering', 'SheetReferenceView') [ECClassId],
```

Comment:
Since this view conceptually extends the persisted definition of a bis.SheetReference, it should return its ECClassId (i.e. [sr].[ECClassId]) rather than the View-class' ECClassId.

Reply:
Done